src/core/qgsapplication.h

```diff
@@ -882,6 +882,8 @@ class CORE_EXPORT QgsApplication : public QApplication
     // ... but in case QgsApplication is never instantiated (eg with custom designer widgets), we fall back to static members
     static ApplicationMembers *sApplicationMembers;
 
+    static QgsAuthManager *sAuthManager;
```

Comment:
why is this not included in `sApplicationMembers` ?

Reply:
not really a new thing - just moved here out of `QgsApplication::authManager` to make it available in `QgsApplication::exitQgis` - corresponding to it's initialization in `QgsApplication::initQgis`. Initializing in `QgsApplication`'s ctor would be too early.

Reply:
thanks for the clarification.